src/test/home/App.test.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import App from '../components/App'
-import HomeFooter from '../components/HomeFooter'
+
+import App from '../../components/home/App'
+import Footer from '../../components/home/Footer'
 
 it('renders without crashing', () => {
   const div = document.createElement('div')
   ReactDOM.render(<App />, div)
 })
 
-it('HomeFooter renders without crashing', () => {
+it('Home Footer renders without crashing', () => {
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```